packages/css-dev-tools/postcssPluginConfig.js

```diff
@@ -43,6 +47,9 @@ const styleLintConfig = require('./styleLintConfig')
 const plugins = [
   stylelint({ config: styleLintConfig }),
   reporter({ clearReportedMessages: true }),
+  addVars({
```

Comment:
where is it ? I don't see the code. 
It would be more versatile to create the `preppend` and be able to just add anything at the top of the file. 
Could be handy.

Reply:
Fixed